extensions/xml/syntaxes/xml.tmLanguage.json

```
@@ -357,9 +357,9 @@
 						"0": {
 							"name": "punctuation.definition.comment.xml"
 						},
-						"end": "--%>",
-						"name": "comment.block.xml"
 					}
```

Doesn't this need a comma 
```suggestion
					},
```